lib/linkify.dart

```diff
@@ -111,8 +136,13 @@ List<LinkifyElement> linkify(
   final emailMatch = linkTypes.contains(LinkType.email)
       ? _linkifyEmailRegex.firstMatch(text)
       : null;
+  final phoneMatch = linkTypes.contains(LinkType.phone)
+      ? phoneRegex != null
```

Comment:
Still here too

Reply:
Really Sorry, i thought i had to preserve the ability to change the phone regex, changing it..